### ISSUE TITLE
LICENSE.md is not in the release package

### ISSUE THREAD

**Issue by rob4acre** [canjs/can-namespace#3]
I used the license-checker package to generate a list of packages and their licenses for my application. I noticed that can-namespace was reporting an UNKNOWN license. After a little investigation it seems to me that the LICENSE.md file which is in this repository is not in the actual 1.0.0 release package that is installed via npm. This doesn't cause me any problems, but I thought that you might like to know.

**Comment by justinbmeyer** [comment]
Thanks for reporting!  Hmmm.  We were never planning on releasing a new version of this.  Maybe we do a `1.0.1` when CanJS 5 comes out?  It might be safe to do in 3 and 4.

**Comment by phillipskevin** [comment]
do we want to do this for 5.0? cc @justinbmeyer @matthewp.

**Comment by matthewp** [comment]
I don't see a point. Why do people need a license file in the package?

**Comment by cmcrothers** [comment]
For auditing of applications so that we can confirm that we are compliant especially for code that may be licensed GPL PR #5

**Comment by justinbmeyer** [comment]
Just to give a bit more context on this ...  

First, everything in DoneJS is MIT licensed.

Second, `can-namespace` is treated as a "global" for all of CanJS's other projects.  It's used to make sure that a user can not have two versions of the same stateful package.  All stateful packages register themselves on this project.

Releasing another version of this package might cause multiple versions to load.  It probably won't, but it likely should be done as part of a major release.  6.0 is the next opportunity.

**Comment by haata** [comment]
Yeah, all automated auditing tools will flag this package and non-compliant. Which sadly requires exception requests for every org that ends up using this package.
I've also floated around re-implementing the tool to just get around the maintenance headache.
